postgresql/resource_postgresql_database.go

```diff
@@ -476,6 +488,54 @@ func setDBOwner(db *DBConnection, d *schema.ResourceData) error {
 	return err
 }
 
+func setAlterOwnership(db *DBConnection, d *schema.ResourceData) error {
+	if d.HasChange(dbOwnerAttr) || d.HasChange(dbAlterObjectOwnership) {
```

Comment:
if only `dbAlterObjectOwnership` changes, it seems you will execute the `REASSIGN` with the same previous and new owner so I'm not sure to see the point  on this case :thinking: 

At least you should check if `currentOwner` and `newOwner` are different below.

Reply:
Thanks for the review. If updated the code so it checks if currentOwner and newOwner are equal before executing the sql statements